funcx_endpoint/funcx_endpoint/endpoint/utils/zhelpers.py

```diff
@@ -0,0 +1,56 @@
+# encoding: utf-8
```

Comment:
`funcx_endpoint/endpoint/utils/*` should be gone. I suspect that this must've gotten back in from a bad merge.